### DISCUSSION TITLE
Reminder to register for User Group Meeting

### DISCUSSION THREAD

**Discussion by surfing-in-the-sun** [sdcTools/UserSupport#208] [User Group]
Dear Members, 

we would like to remind you to register for the **User Group Meeting next week 
21.01.2021 (10am -12:30pm)** [here](https://sdctools.github.io/UserSupport/) .

Registered Users will receive an E-Mail with the Link for the online meeting.

Users who can't attend the meeting, will still be able to view the report of the meeting and the shown presentations afterwards here on Github.

We also want to draw your attention to the **Online Course on Statistical Disclosure Control (SDC)** that will be held by EUROSTAT in March. The **deadline for registration  is on 26.01.2021** and you can find all information [here](https://ec.europa.eu/eurostat/cros/content/statistical-disclosure-control-1_en) .

Best regards, Paula

**Comment by Wim-Kloek** [comment]
Dear Paula,
I just received three messages from the GitHub environment (one time number 2 for user-support, one time number 2 for user-group and one time number 3 for user-group). I hope that was what you had in mind.
Wim
